tls/s2n_connection.h

```diff
@@ -47,6 +47,11 @@
 
 #define S2N_PEER_MODE(our_mode) ((our_mode + 1) % 2)
 
+#define S2N_SERIALIZED_CONN_FIXED_SIZE (8 + S2N_TLS_PROTOCOL_VERSION_LEN + S2N_TLS_CIPHER_SUITE_LEN \
+        + S2N_TLS_SEQUENCE_NUM_LEN + S2N_TLS_SEQUENCE_NUM_LEN + 2)
+#define S2N_SERIALIZED_CONN_TLS12_SIZE (S2N_SERIALIZED_CONN_FIXED_SIZE + S2N_TLS_SECRET_LEN \
+        + S2N_TLS_RANDOM_DATA_LEN + S2N_TLS_RANDOM_DATA_LEN)
```

Comment:
These defines are more involved now-- should we have a s2n_handshake_serialize.h? Or maybe they should be methods instead / be calculated in s2n_connection_serialization_length?